pages/apim/3.x/event-native-apim/event-native-apim-documentation.adoc

```diff
@@ -0,0 +1,241 @@
+[[event-native-apim-documentation]]
+= Documentation
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/event_native_apim_documentation.html
+:page-folder: apim/event-native-apim
+:page-layout: apim3x
+
+[label label-version]#New in version 3.20.0#
+[label label-version]#BETA release#
+
+[NOTE]
+====
+This document assumes you are familiar with synchronous APIs, asynchronous APIs, and the OpenAPI specification.
+====
+
+[TIP]
+====
+Before running any of the examples on this page, work through some of the  link:event_native_apim_example_use_cases/html[example use cases] so you know that the event-native engine is running.
+====
+
+== Asynchronous APIs
+
+In this release, you can use the gateway to connect to asynchronous APIs. You can configure asynchronous API connections using the RESTful interface to the management API.
+
+The OpenAPI specification for the management API is available at link:/apim/3.x/management-api/3.19/swagger.json[`swagger.json`] (link:/apim/3.x/apim_installguide_rest_apis_documentation.html#apim_console_api_reference[reference documentation]). Examples of how to use the management API to configure asynchronous APIs can be found in the link:https://github.com/gravitee-io/postman-collections[Gravitee Postman Collections], which are described in link:event_native_apim_example_use_cases.html[example use cases].
+
+[NOTE]
+====
+The management API is in beta and it may change.
+====
+
+== Entrypoints and endpoints
+
+An _entrypoint_ is how an end user connects to the Gravitee gateway. An _endpoint_ is how the Gravitee gateway connects to the upstream API.
+
+The following types of entrypoint are available.
+
+* SSE
+* SSE advanced
+* Websockets
+* HTTP GET
+* HTTP POST
+* Webhook
+* Webhook advanced (EE)
+
+The following types of endpoint are available
+
+* Kafka
+* Kafka advanced (EE)
+* MQTT
+* MQTT advanced (EE)
+
+== Create a v4 async API 
```

Comment:
This is for me too technical and straighforward. We need to explain what have changed, what is a listener, what is an entrypoint, what is an endpoint, what is the new flow, how they interact